tests/e2e/utils/helpers.go

```diff
@@ -136,6 +172,16 @@ func HTTPGetRawNTimes(url string, n int) (*http.Response, error) {
 	return res, err
 }
 
+// httpGetRaw is a helper to make GET request call to url
+func httpGetRaw(client http.Client, url string) (*http.Response, error) {
```

Comment:
modify function `HTTPGetRaw` to use this function?

Reply:
Done